resources/docs/introduction.md

```diff
@@ -20,6 +20,19 @@ Greetings {{ $friend }}, let's build Chirper with Blade!
     <span class="w-full inline-flex items-center justify-center self-stretch px-4 py-2 text-sm text-red-600 text-center font-bold uppercase bg-white dark:bg-dark-700 ring-1 ring-red-600 ring-offset-1 dark:ring-offset-dark-600 transform transition-transform group-hover:-translate-y-1 group-hover:-translate-x-1 group-focus:-translate-y-1 group-focus:-translate-x-1">Build Chirper with Blade</span>
 </a>
 
+### Livewire
```

Comment:
I wonder whether it's worth mentioning Volt here too? It was only later on that I realised which "flavour" of Livewire I'd be using. We could also link to the Volt docs in the paragraph below so people have a reference point for what they'll be working with.

Reply:
I don't think so. Having single file components is a Livewire feature, and I will explain that bellow already.